v22.1/aws-dms.md

```diff
@@ -0,0 +1,89 @@
+- Ensure you have a secure, publicly available CockroachDB cluster running v22.1.
+
+## Step 1. Create a new `dms` user
+
+1. [Connect to your {{ site.data.products.serverless }} cluster](../cockroachcloud/connect-to-a-serverless-cluster.html).
+
+1. In the SQL client, create a new `dms` admin user to handle the migration. Replace `'<password>'` with a strong password:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    > CREATE USER dms WITH PASSWORD '<password>';
+    > GRANT admin TO dms;
+    > ALTER USER dms SET expect_and_ignore_not_visible_columns_in_copy = true;
+    ~~~
+
+    {{site.data.alerts.callout_danger}}
+    Do not use this user for normal SQL activity. The `expect_and_ignore_not_visible_columns_in_copy` session variable may make it behave unpredictably for normal usage.
+    {{site.data.alerts.end}}
+
+## Step 2. Create a target endpoint pointing to CockroachDB
+
+1. In the AWS Console, open **AWS DMS**.
+1. Open **Endpoints** in the sidebar. A list of endpoints should show, if any exist.
+1. In the top-right portion of the window, select **Create endpoint**.
+1. In the **Endpoint type** section, select **Target endpoint**.
+1. Supply an **Endpoint identifier** to identify the new target endpoint.
+1. For the **Source engine** dropdown, select **PostgreSQL**.
+1. For the **Access to endpoint database** radio button, select the **Provide access information manually**.
+1. Enter in the **Server name**, **Port**, **User name**, **Password**, and **Database name** of your CockroachDB cluster. For **Secure Socket Layer (SSL) mode**, select NEED VALUE HERE
+1. You can test the connection if needed under **Test endpoint connection (optional)**. You may need to upload a certificate under **Certificates** if using one.
+
+## Step 3. Create a replication task
+
+### Step 3.1. Task configuration
+
+1. While in **AWS DMS**, select **Database migration tasks** in the sidebar. A list of database migration tasks should show, if any exist.
+1. In the top-right portion of the window, select **Create task**.
+1. Supply a **Task identifier** to identify the replication task.
+1. Select the **Replication instance** and **Source database endpoint** you created prior to starting this tutorial.
+1. For the **Target database endpoint** dropdown, select the CockroachDB endpoint created in the previous section.
+
+### Step 3.2. Task settings
+
+1. For the **Editing mode** radio button, keep **Wizard** selected.
+1. For the **Target table preparation mode**, keep **Drop tables on target** selected.
+1. (Optional) You can **Enable CloudWatch logs** for extra insight about the replication.
+1. Switch the **Editing mode** from **Wizard** to **JSON editor**.
+1. Locate the `BatchApplyEnabled` setting and change its value to `true`. Information about the `BatchApplyEnabled` setting can be found [here](https://aws.amazon.com/premiumsupport/knowledge-center/dms-batch-apply-cdc-replication/).
```

Comment:
`BatchApplyEnabled` is optional but recommended. As mentioned above, does not work for "Drop tables on target".

Reply:
Done.